src/main.c

```diff
@@ -222,6 +223,14 @@ struct listener_config_t {
          */
         struct listener_config_t *sibling;
     } quic;
+
+    struct {
+        h2o_url_t client;
```

Comment:
```suggestion
        h2o_url_t url;
```
Maybe. "client" is confusing in sense that it does not imply what property of the client the variable is. In contrast, there is only one "url" in the context of reverse HTTP establishment.

Reply:
Renamed in https://github.com/h2o/h2o/pull/3405/commits/e1ab7d0b0850849a35d70acd2d3d0cdb95878abe, but do you also prefer to rename the `client` configuration directive under `listen` ?

Reply:
> do you also prefer to rename the client configuration directive under listen ?

That might make things less confusing, I think (we know the terms "client" and "server" is confusing due to us reversing the roles).

Reply:
Done in https://github.com/h2o/h2o/pull/3405/commits/a10d7bc3407f99cf4e10d50cf3df769d46dc56d1